app/build.gradle

```diff
@@ -18,8 +18,8 @@ android {
         applicationId "io.bloco.circles"
         minSdkVersion 23
         targetSdkVersion 31
-        versionCode 11
-        versionName "1.0.1"
+        versionCode 12
```

Comment:
Maybe update the CHANGELOG as well if you can (@sdsantos)
https://github.com/blocoio/stacks-circles-app/blob/main/CHANGELOG.md